### ISSUE TITLE
Cannot install requirements.txt

### ISSUE THREAD

**Issue by doguitar** [simse/chronos#2]
- Installed docker container with command `docker run -d --name='chronos' --net='bridge' -e TZ="America/Los_Angeles" -e HOST_OS="Unraid" -p '5001:5000/tcp' -v '<path_to_docker>/chronos':'/chronos':'rw' 'simsemand/chronos'`
- Added new script
- Added line to requirements.txt
- Clicked `Install Pip Requirements`
- Toast message shows `Couldn't install Pip requirements`

Log as follows.
```::ffff:192.168.1.50 - - [2019-04-23 17:56:04] "GET /api/script/shreddit-script/install_requirements HTTP/1.1" 500 443 0.001683
::ffff:192.168.1.50 - - [2019-04-23 17:56:04] "GET /api/scripts HTTP/1.1" 200 328 0.002938
::ffff:192.168.1.50 - - [2019-04-23 17:56:06] "GET /api/scripts HTTP/1.1" 200 328 0.002594
::ffff:192.168.1.50 - - [2019-04-23 17:56:08] "GET /api/scripts HTTP/1.1" 200 328 0.002664
[2019-04-23 17:56:10,063] ERROR in app: Exception on /api/script/shreddit-script/install_requirements [GET]

Traceback (most recent call last):
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
response = self.full_dispatch_request()
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
rv = self.handle_user_exception(e)
File "/usr/local/lib/python3.7/site-packages/flask_restful/__init__.py", line 269, in error_router
return original_handler(e)
File "/usr/local/lib/python3.7/site-packages/flask_cors/extension.py", line 161, in wrapped_function
return cors_after_request(app.make_response(f(*args, **kwargs)))
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
reraise(exc_type, exc_value, tb)
File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
raise value
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
rv = self.dispatch_request()
File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
return self.view_functions[rule.endpoint](**req.view_args)
File "/app/chronos/chronos/web.py", line 85, in install_requirements
'response': chronos.script.Script(uid).install_requirements()
File "/app/chronos/chronos/script.py", line 60, in install_requirements
process = subprocess.check_output(['bash', self.install_requirements_path], universal_newlines=True)
NameError: name 'subprocess' is not defined```

**Comment by simse** [comment]
Hi, thank you for reporting this issue. Could you post your requirements.txt?

**Comment by simse** [comment]
Actually, scratch that. I reproduced the issue, and will be pushing an update ASAP.

**Comment by simse** [comment]
Just pushed a fix. You can force update in Docker view in Unraid.

**Comment by doguitar** [comment]
Thanks but it doesn't seem to be working still. I get no errors or log output though.

my requirements.txt is 
```shreddit```
and my script is
```import shreddit```

executing the script after installing requirements (shows no error) fails with the following exception:

```Traceback (most recent call last):
  File "/chronos/scripts/shreddit-script/shreddit-script.py", line 1, in <module>
    import shreddit
ModuleNotFoundError: No module named 'shreddit'```

**Comment by simse** [comment]
I can't seem to reproduce the issue. Could you, to begin with, force update the container in Unraid so you're certain you're running the latest version? Then could you go to your Chronos folder, and open 'shreddit-script' and check that your requirements.txt is saved correctly. After that open '.venv' in that same folder, then lib -> python3.7 -> site-packages and check that there is a folder called 'shreddit'.

**Comment by doguitar** [comment]
Not sure what the issue was but it seems to be working now. I did do the force update (but nothing was updated but maybe the restart helped) and while that was going, viewed the written requirements.txt which was blank. Then I went into the website, added shredded to requirements.txt and hit save (the save button didn't used to show up when I edited requirements.txt, I don't know why). Checked the file system's requirements.txt and it was correct and then everything worked as expected.

Thanks for the fix/help. Not sure what the issue was. I am running ublock origin and yesterday, I disabled it on all local websites so perhaps that was the issue.

Thanks again.
